microreactors/gcmr/airjacket/sample_slurm.batch

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#SBATCH --nodes=6
+#SBATCH --ntasks-per-node=36
+#SBATCH --time=1:0:00
+#SBATCH -p <queue type>i
+#SBATCH --output=logfile
+#SBATCH --mail-user=<user_email>
+
```

Comment:
which machine does this run on?
the I_MPI stuff looks pretty specific

Reply:
IMO I think the job script can be removed altogether. It's going to be very specific to the particular cluster (e.g. slurm, vs pbs vs ...). Also I think most (if not all?) of the other VTB cases do not provide job scripts.